packages/terra-site/src/examples/content-container/ContentContainerFill.jsx

```diff
@@ -1,24 +1,54 @@
 import React from 'react';
 
 import ContentContainer from 'terra-content-container';
-import Button from 'terra-button';
+import Heading from 'terra-heading';
 
-const display1 = <p key="1">display1display1display1display1display1display1display1display1display1display1display1display1display1display1</p>;
-const display2 = <p key="2">display2display2display2display2display2display2display2display2display2display2display2display2display2display2</p>;
-const display3 = <p key="3">display 3</p>;
-const display4 = <p key="4">display 4</p>;
-const display5 = <p key="5">display 5</p>;
-const display6 = <p key="6">display 6</p>;
+const display1 = (<p key="1">
```

Comment:
Nit: can we reduce the number of displays to 2 or 3. The example's pretty long for fake input that is repeated.

Reply:
Updated https://github.com/cerner/terra-core/commit/0f1aebd45306d036c18a279505d8d0aeef752712

Reply:
Thank you!